### ISSUE TITLE
gcloud beta emulators spanner start broken in 1.1.0

### ISSUE THREAD

**Issue by tengis617** [GoogleCloudPlatform/cloud-spanner-emulator#7]
running `gcloud beta emulators spanner start` produces the following output:
```
$ gcloud beta emulators spanner start
Executing: docker run -p 127.0.0.1:9010:9010 -p 127.0.0.1:9020:9020 --noenable_fault_injection gcr.io/cloud-spanner-emulator/emulator:1.1.0
[cloud-spanner-emulator] unknown flag: --noenable_fault_injection
[cloud-spanner-emulator] See 'docker run --help'.
```

local gcloud env:
```
$ gcloud version
Google Cloud SDK 313.0.0
app-engine-go 
app-engine-python 1.9.91
beta 2020.10.02
bq 2.0.61
cloud-datastore-emulator 2.1.0
core 2020.10.02
gsutil 4.53
```

**Comment by Vizerai** [comment]
This will be updated soon. In the meantime you can run the command without the flag.

**Comment by alapkarapurkar** [comment]
This is fixed in gcloud version 314.0.0
